src/args/default.nix

```diff
@@ -16,7 +16,9 @@ let
     __shellCommands__ = ./shell-commands/makes-setup.sh;
     __shellOptions__ = ./shell-options/makes-setup.sh;
     __toModuleOutputs__ = import ./to-module-outputs/default.nix args;
-    asBashArray = args: "( ${lib.strings.escapeShellArgs args} )";
+    asBashArray = name: list: builtins.toFile
+      name
+      "local ${name}=( ${lib.strings.escapeShellArgs list} )";
```

Comment:
if it's logner than one line please move it to a file under src/args/__name__/default.nix